docs/readthedocs/source/doc/Nano/Overview/pytorch_inference.md

```diff
@@ -172,7 +193,7 @@ trainer.predict(ov_q_model, dataloader)
 Same as using ONNXRuntime accelerator, it equals to converting the model from PyTorch to OpenVINO firstly and then doing quantization on the converted OpenVINO model:
 ```python
 ov_model = InferenceOptimizer.trace(model, accelerator='openvino', input_sample=x):
```

Comment:
remove trailing ':'

Reply:
have removed related part of "trace the quantize".